plugins/modules/packaging/os/pkgin.py

```diff
@@ -255,13 +267,16 @@ def install_packages(module, packages):
     install_c = 0
 
     for package in packages:
-        if query_package(module, package):
+        query_result = query_package(module, package)
+        if query_result is PackageState.INSTALLED:
```

Comment:
```suggestion
        if query_result == PackageState.INSTALLED:
```
Since you also used that above. Same for below.

Reply:
Actually I have removed the INSTALLED state since it would require usage of a logical operator and that somehow I don't like tbh.